### PR TITLE
[NLU-4079] Fix for text numerals with "point" and "x billions and y millions"

### PR DESCRIPTION
**Pull request by KanchanKumar12**
The bug has a use-case "six point five billion". However, as confirmed with native speakers, most bank contact center uses text like "6 billion and 500 millions"

This PR fixes two use-cases (x and y are text numerals):
1. x point y (e.g. six point eight) 
2. x billion and y million (e.g. three billion and five hundred million)

Bug: NLU-4079

### CODE REVIEW

File: Python/libraries/recognizers-number/recognizers_number/resources/arabic_numeric.py
Comment (KanchanKumar12) [inline Python/libraries/recognizers-number/recognizers_number/resources/arabic_numeric.py:1]: My OCD kicked in. I had to sort the keys in maps so it is easy to read their regexes

**Comment by KanchanKumar12** [comment]
@richcamp 
For use-case "6 billion and 500 millions", NLU-4078 fix will introduce "RoundMultiplierRegex" in Arabic numeric regex. It may be fixed as a part of it. However, can't say it for sure if it is going to be fixed even after introducing the above mentioned regex. I would like to know if this use case is a common input text that we can expect from Arabic customers. If it is, we will create a separate bug to look into it after NU-4078. Thanks!

**Comment by KanchanKumar12** [comment]
Pipeline run: https://jenkins.ininica.com/job/NLU/job/Recognizers-Text/421/

File: Python/libraries/recognizers-number/recognizers_number/resources/arabic_numeric.py
Comment (andrew-gradinari) [reply Python/libraries/recognizers-number/recognizers_number/resources/arabic_numeric.py:1]: Nice, well done 👍

**Review by andrew-gradinari — APPROVED** [review]

**Review by colm-dillon — APPROVED** [review]

File: Python/libraries/recognizers-number/recognizers_number/resources/arabic_numeric.py
Comment (samhickey25) [reply Python/libraries/recognizers-number/recognizers_number/resources/arabic_numeric.py:1]: thank you!

**Review by samhickey25 — APPROVED** [review]

**Pull request merged**